modules/apps/27-interchain-accounts/controller/types/msgs_test.go

```diff
@@ -0,0 +1,76 @@
+package types_test
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/controller/types"
+	icatypes "github.com/cosmos/ibc-go/v5/modules/apps/27-interchain-accounts/types"
+	ibctesting "github.com/cosmos/ibc-go/v5/testing"
+)
+
+var (
+	testAccAddress     = "cosmos17dtl0mjt3t77kpuhg2edqzjpszulwhgzuj9ljs"
```

Comment:
super nit but maybe we can add this a const file?

Reply:
I've added `TestAccAddress` to testing/values.go